### PR TITLE
Keyboard exit ethnio

### PR DESCRIPTION
**Pull request by noahmanger**
Adds a very crude hack to allow the Ethnio popup to be closed by keyboard. Because both the pre-screener modal and the screener modal are closed by `<div>`s masquerading as buttons, it took some ugly code. 

First, it adds a tabindex to the initial close button on the pre-screener modal and fires a click event when hitting enter on it.

Second, if the actual screener is opened, it adds a listener to the body to listen for `escape` and then simply removes the iframe. 

I just heard back from Ethnio that they're looking into this accessibility issue, but in the mean time this can work.

Resolves https://github.com/18F/openFEC-web-app/issues/762

### CODE REVIEW

File: fec/home/templates/home/home_page.html [hunk 1]
```diff
@@ -102,5 +102,34 @@ <h2 class="feature__title">The FEC API: Use our data in your project</h2>
...
+            if (e.keyCode === 13) {
+              $(this).click();
+              observer.disconnect();
+            }
+          });
+        }
+        if ( mutation.addedNodes[0].id === 'ethnio-screener-id-70862' ) {
+          $('body').on('keyup', function(e){
+            if (e.keyCode === 27) {
+              $('iframe').remove();
```
Comment (jmcarp) [inline fec/home/templates/home/home_page.html]: There can be other iframes on the page--for example, the wagtail admin widget. This isn't a big deal, but if it's simple, could we remove only the ethnio iframe here?

Comment (noahmanger) [inline fec/home/templates/home/home_page.html]: I ended up reaching out to Ethnio to see if they could just fix this themselves and it seems like they may have. I'll double check, but this is likely unnecessary n ow.

Comment (jmcarp) [inline fec/home/templates/home/home_page.html]: :tada:

**Comment by noahmanger** [comment]
Ok, Ethnio updated their script to close the popups on escape. Still not tabbable, but they said they're working on that. Just going to close this.

**Pull request closed without merging**